manifests/configmap.yaml

```diff
@@ -3,8 +3,9 @@ kind: ConfigMap
 metadata:
   name: postgres-operator
 data:
-  # assumes the ns exists before the operator starts
-  # the env var with the same name may overwrite it in the operator pod
+  # the env var with the same name in the operator pod may overwrite this value
+  # if neither is set, listen to the 'default' namespace
+  # if set to the empty string "", listen to all namespaces
```

Comment:
We should listen to the operator namespace either when the parameters are empty or not present, and only listen to all namespaces when we have a '*'.